tests/util_test.py

```diff
@@ -54,3 +55,13 @@ def test_postcode_to_itl1_post_code_area_not_exist_raises_error():
         postcode_to_itl1("ZZ1 2AB")
 
     assert exc_info.value.args[0] == 'Postcode Area "ZZ" is invalid and has no mapping.'
+
+
+def seed_test_database(model, model_data):
```

Comment:
This could do with a docstring really, sorry, it was me that committed the func without one. I would add now but don't want to conflict with your local branches. I can still do it if you'd prefer, let me know :)